src/org.ts

```diff
@@ -140,3 +140,19 @@ export async function getOrg(
     },
   };
 }
+
+export async function deleteOrg(
+  name: string,
+  options: ServiceOptions,
+): Promise<Result<undefined, OrgProblemType>> {
+  const orgModel = getModelForClass(OrgModelSchema, {
+    existingConnection: options.dbConnection,
+  });
+  await orgModel.deleteOne({
+    name,
+  });
+
+  return {
```

Comment:
Log, please. Also make sure to update the existing happy-path test.

Reply:
I thought this would be a better way of logging. To check if it actually deleted something. If yes then I think we could update the "updateOrg" method to check if it really updated something too